pyriemann/utils/covariance.py

```diff
@@ -6,36 +6,85 @@
 
 from .test import is_square
 
-# Mapping different estimator on the sklearn toolbox
 
+def _fpcm(X, *, init=None, tol=10e-3, n_iter_max=50, assume_centered=False):
+    """Fixed point covariance estimator.
 
-def _lwf(X):
+    Parameters
+    ----------
+    X : ndarray, shape (n_channels, n_times)
+        Multi-channel time-series.
+    init : None | ndarray, shape (n_channels, n_channels), default=None
+        A matrix used to initialize the algorithm.
+        If None, the normalized sample covariance matrix is used.
+    tol : float, default=10e-3
+        The tolerance to stop the fixed point estimation.
+    n_iter_max : int, default=50
+        The maximum number of iterations.
+    assume_centered : bool, default=False
+        If `True`, data will not be centered before computation.
+        Useful when working with data whose mean is almost, but not exactly
+        zero. If `False`, data will be centered before computation.
+
+    Returns
+    -------
+    cov : ndarray, shape (n_channels, n_channels)
+        Fixed point covariance matrix.
```

Comment:
can you add the ref "Theoretical analysis of an improved covariance matrix estimator in non-Gaussian noise" here?

Do I understand correctly that this does some kind of robust estimation with M-estimators?

Reply:
Done.

It is not a robust M-estimator. M is just the name of the covariance matrix.

Reply:
You were right: fixed point covariance estimates the distribution-free robust M-estimator, see [Tyler1987](https://projecteuclid.org/journals/annals-of-statistics/volume-15/issue-1/A-Distribution-Free-M-Estimator-of-Multivariate-Scatter/10.1214/aos/1176350263.full). 
I have updated doc.